<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgTextures</class>
 <widget class="QDialog" name="CQDlgTextures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Texture Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Texture relative position/orientation</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqAlpha">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Beta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqBeta">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqGamma">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqY">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqZ">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqX">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>35</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Texture scaling</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Along V</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqV">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Along U</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqU">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Other texture properties</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="qqInterpolate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Interpolate colors</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqModulate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Modulate colors</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqRepeatU">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Repeat along U</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqRepeatV">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Repeat along V</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qqMapMode">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>18</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Mapping mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="qqRemoveSelect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Remove texture</string>
   </property>
  </widget>
  <widget class="QToolButton" name="qqLoad">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Load new texture</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>205</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Texture info</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Associated texture coordinates:</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqTextureName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>blabla</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Associated texture:</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqTextureCoordinates">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>blabla</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqRemoveSelect</tabstop>
  <tabstop>qqLoad</tabstop>
  <tabstop>qqAlpha</tabstop>
  <tabstop>qqBeta</tabstop>
  <tabstop>qqGamma</tabstop>
  <tabstop>qqX</tabstop>
  <tabstop>qqY</tabstop>
  <tabstop>qqZ</tabstop>
  <tabstop>qqU</tabstop>
  <tabstop>qqV</tabstop>
  <tabstop>qqMapMode</tabstop>
  <tabstop>qqInterpolate</tabstop>
  <tabstop>qqModulate</tabstop>
  <tabstop>qqRepeatU</tabstop>
  <tabstop>qqRepeatV</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
